pkg/selinux/setroubleshoot-client.js

```diff
@@ -111,7 +112,10 @@ client.init = function() {
           doText
           analysisId: plugin id. It can be used in org.fedoraproject.SetroubleshootFixit.run_fix()
           fixable: True when an alert is fixable by a plugin
-          reportBug: True when an alert should be reported to bugzilla
+          reportBug: True when an alert should be reported
+      firstSeen: when the alert was seen for the first time, iso8601 format is used - '%Y-%m-%dT%H:%M:%SZ'
+      lastSeen: when the alert was seen for the last time, iso8601 format is used - '%Y-%m-%dT%H:%M:%SZ'
+      level: "green", "yellow" or "red"
```

Comment:
> This is part of the released setroubleshoot API, it's a string coming from python. I didn't want to change this for now, until we've put more thought and work into localization. Should we try to change the setroubleshoot API? Or parse the string here before passing it on, only to format it into a string again?

I think we should try to change their API if it's not stable yet.

At least we should put something more readable into our UI.